common/proc_allowing-external-access-knative-services-OCP-4x.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+
+[id='allowing-external-access-knative-services-OCP-4x_{context}]
+= Allowing external access to Knative services on OCP 4.0
+
+Use a configured hostname to access Knative services on the OpenShift 4.0 cluster
+
+.Procedure
+
+. Updating Knative's config-domain ConfigMap and replacing "example.com" with your domain's suffix.
+
+NOTE: This will not be needed once the new operator capabilities are enabled to set the value automatically.
```

Comment:
Remove note - we shouldn't mention any functionality we plan to add in the future.